UnitsNet/CustomCode/UnitSystem.cs

```diff
@@ -411,6 +411,33 @@ public string[] GetAllAbbreviations(Type unitType, int unitValue)
                 : GetCached(FallbackCulture).GetAllAbbreviations(unitType, unitValue);
         }
 
+        /// <summary>
+        ///     Get all abbreviations for unit.
+        /// </summary>
+        /// <param name="unitType">Enum type for unit.</param>
+        /// <param name="unitValue">Enum value for unit.</param>
+        /// <returns>Unit abbreviations associated with unit.</returns>
+        [PublicAPI]
+        public string[] GetAllAbbreviations(Type unitType)
+        {
+            Dictionary<int, List<string>> unitValueToAbbrevs;
+            List<string> abbrevs = new List<string>();
+
+            if (_unitTypeToUnitValueToAbbrevs.TryGetValue(unitType, out unitValueToAbbrevs))
+            {
+                foreach (var item in unitValueToAbbrevs)
```

Comment:
It is giving me a compile time error trying to do this. I think it is because `unitValueToAbbrevs.Values` is a collection of lists. Is there another way with LINQ that I have not found?